### PR TITLE
add BinPastes to pastebins

### PR DESCRIPTION
**Pull request by querwurzel**
https://github.com/querwurzel/BinPastes

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/binpastes.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: BinPastes
+website_url: https://github.com/querwurzel/BinPastes
+source_code_url: https://github.com/querwurzel/BinPastes
+description: Minimal pastebin also supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment.
```
Comment (TheGiraffe3) [inline software/binpastes.yml:4]:
```suggestion
description: "Minimal pastebin also supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment."
```

File: software/binpastes.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: BinPastes
+website_url: https://github.com/querwurzel/BinPastes
+source_code_url: https://github.com/querwurzel/BinPastes
+description: "Minimal pastebin also supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment."
```
Comment (Ki-er) [inline software/binpastes.yml:4]: Grammar change
```suggestion
description: "Minimal pastebin supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment."
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Very quick change

File: software/binpastes.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+website_url: https://github.com/querwurzel/BinPastes
+source_code_url: https://github.com/querwurzel/BinPastes
+description: "Minimal pastebin supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment."
+licenses:
+  - Apache-2.0
+platforms:
+  - Java
+tags:
+  - Pastebins
+depends_3rdparty: true
```
Comment (nodiscc) [inline software/binpastes.yml:11]:
```suggestion
depends_3rdparty: true
```

Why does this apply?

Comment (querwurzel) [reply software/binpastes.yml:11]: Depends on what counts as 3rd party. You need a runtime environment (JDK) and the software uses libraries.

Comment (nodiscc) [reply software/binpastes.yml:11]: I see. It does not apply then, please remove this line.

> https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md
> whether the software depends on a third-party service outside the user's control

Comment (querwurzel) [reply software/binpastes.yml]: Allright, set it to false

File: software/binpastes.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+website_url: https://github.com/querwurzel/BinPastes
+source_code_url: https://github.com/querwurzel/BinPastes
+description: "Minimal pastebin supporting client-side encryption, fulltext search, one-time messages. Intended for one to few users looking for a simple pastebin deployment."
+licenses:
+  - Apache-2.0
+platforms:
+  - Java
+tags:
+  - Pastebins
+depends_3rdparty: false
```
Comment (nodiscc) [inline software/binpastes.yml:11]:
```suggestion
```

false is already the default value, omit it completely

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**